celery_once/helpers.py

```diff
@@ -27,14 +28,20 @@ def now_unix():
     return int(time())
 
 
+def items_sorted_by_key(kwargs):
+    return sorted(six.iteritems(kwargs), key=operator.itemgetter(0))
+
+
 def force_string(kwargs):
     """
     Force key in dict or list to a string.
     Fixes: https://github.com/TrackMaven/celery-once/issues/11
     """
     if isinstance(kwargs, dict):
-        return {
-            force_string(key): force_string(value) for key, value in six.iteritems(kwargs)}
+        return [
+            (force_string(key), force_string(value))
```

Comment:
Would change this to use OrderedDict (ref: https://stackoverflow.com/questions/9001509/how-can-i-sort-a-dictionary-by-key)